cfme/scripting/conf.py

```diff
@@ -101,5 +103,27 @@ def decrypt(conf_name, delete, skip):
     print('{} conf decrypted'.format(conf_name))
 
 
+@main.command("write", help="Write a username and password into vault")
+@click.argument("env")
+@click.argument("key")
+@click.argument("username")
+@click.argument("password")
+def write(env, key, username, password):
+    with settings.using_env(env):
+        data = settings.as_dict()
+        data[key.upper()] = {"username": username, "password": password}
+        vault_loader.write(settings, data)
+    print(f"Username '{username}' and password written to '{env}' successfully")
+
+
+@main.command("read", help="Write a username and password into vault")
+@click.argument("env")
+@click.argument("key")
+def read(env, key):
+    with settings.using_env(env):
+        print(settings.as_dict()[key.upper()])
+        return settings.as_dict()[key.upper()]
```

Comment:
When we fetch keys from vault with dynaconf it is always upper case. So its better to store/read it with `.upper()`